app.py

```diff
@@ -1,14 +1,20 @@
 import os
 from flask import Flask
 from api import api
+from flask_cors import CORS
 
 app = Flask(__name__)
+
 # load default config
 app.config.from_object('config')
 # load override config if exists
 if 'APP_CONFIG' in os.environ:
     app.config.from_envvar('APP_CONFIG')
 api.init_app(app)
+# enable CORS if flag set in config
+if os.getenv('CORS_ENABLE') == 'true':
+    CORS(app, origins='*')
```

Comment:
Maybe you could print something out here to let people know CORS is enabled? Just a thought, not sure if it is necessary or not. Other than that this looks good.

Reply:
I added some colored output so it stands out.. but I can change it back to white if that's preferable.

Reply:
also, I put the output in backend.py since the other output was located there, but it could also live in `app.py` if you wanted all the CORS-related stuff in one place.

Reply:
`backend.py` is intended to encapsulate model-specific code and should in theory be independent of the API front-end layer. So this would best be placed in `app.py` rather

Reply:
That makes perfect sense.. I've gone ahead and moved it over. Thank you all for the patience

Reply:
The color coding doesn't really provide much benefit. Other than that I think this is good to go.